packages/core-app-api/report.api.md

```diff
@@ -640,6 +640,9 @@ export class SamlAuth
 // @public
 export type SignInPageProps = PropsWithChildren<{
   onSignInSuccess(identityApi: IdentityApi): void;
+  ErrorComponent?: ComponentType<{
+    error?: Error;
+  }>;
```

Comment:
No need to add this here right? these are the required props from the sign-in page from the app's PoV. Define it as an extra prop for the implementations in `core-components` instead